subworkflows/local/pair_variant_calling.nf

```diff
@@ -73,71 +66,11 @@ workflow PAIR_VARIANT_CALLING {
-            BGZIP_VC_MANTA_SOMATIC(MANTA_SOMATIC.out.somatic_sv_vcf)
-
-            manta_sv_vcf_to_concat = BGZIP_VC_MANTA_SV.out.vcf.map{ meta, vcf ->
-                new_meta = meta.clone()
-                new_meta.id = new_meta.tumor_id + "_vs_" + new_meta.normal_id
-                [new_meta, vcf]
-            }.groupTuple(size: num_intervals)
-
-            manta_small_indels_vcf_to_concat = BGZIP_VC_MANTA_SMALL_INDELS.out.vcf.map{ meta, vcf ->
-                new_meta = meta.clone()
-                new_meta.id = new_meta.tumor_id + "_vs_" + new_meta.normal_id
-                [new_meta, vcf]
-            }.groupTuple(size: num_intervals)
-
-            manta_diploid_vcf_to_concat = BGZIP_VC_MANTA_DIPLOID.out.vcf.map{ meta, vcf ->
-                new_meta = meta.clone()
-                new_meta.id = new_meta.tumor_id + "_vs_" + new_meta.normal_id
-                [new_meta, vcf]
-            }.groupTuple(size: num_intervals)
-
-            manta_somatic_sv_vcf_to_concat = BGZIP_VC_MANTA_SOMATIC.out.vcf.map{ meta, vcf ->
-                new_meta = meta.clone()
-                new_meta.id = new_meta.tumor_id + "_vs_" + new_meta.normal_id
-                [new_meta, vcf]
-            }.groupTuple(size: num_intervals)
-
-            CONCAT_MANTA_SV(manta_sv_vcf_to_concat, fasta_fai, intervals_bed_combine_gz)
-            CONCAT_MANTA_SMALL_INDELS(manta_small_indels_vcf_to_concat,fasta_fai, intervals_bed_combine_gz)
-            CONCAT_MANTA_DIPLOID(manta_diploid_vcf_to_concat, fasta_fai, intervals_bed_combine_gz)
-            CONCAT_MANTA_SOMATIC(manta_somatic_sv_vcf_to_concat, fasta_fai, intervals_bed_combine_gz)
-
-            manta_candidate_small_indels_vcf = CONCAT_MANTA_SV.out.vcf
-            manta_candidate_sv_vcf           = CONCAT_MANTA_SMALL_INDELS.out.vcf
-            manta_diploid_sv_vcf             = CONCAT_MANTA_DIPLOID.out.vcf
-            manta_somatic_sv_vcf             = CONCAT_MANTA_SOMATIC.out.vcf
-
-            ch_versions = ch_versions.mix(BGZIP_VC_MANTA_SV.out.versions)
-            ch_versions = ch_versions.mix(BGZIP_VC_MANTA_SMALL_INDELS.out.versions)
-            ch_versions = ch_versions.mix(BGZIP_VC_MANTA_DIPLOID.out.versions)
-            ch_versions = ch_versions.mix(BGZIP_VC_MANTA_SOMATIC.out.versions)
-
-            ch_versions = ch_versions.mix(CONCAT_MANTA_SV.out.versions)
-            ch_versions = ch_versions.mix(CONCAT_MANTA_SMALL_INDELS.out.versions)
-            ch_versions = ch_versions.mix(CONCAT_MANTA_DIPLOID.out.versions)
-            ch_versions = ch_versions.mix(CONCAT_MANTA_SOMATIC.out.versions)
-
-        }
-
-        manta_vcf = manta_vcf.mix(manta_candidate_small_indels_vcf,manta_candidate_sv_vcf,manta_diploid_sv_vcf,manta_somatic_sv_vcf)
+        RUN_MANTA_SOMATIC(cram_pair_intervals_gz_tbi,
+                        fasta,
+                        fasta_fai,
+                        num_intervals,
+                        intervals_bed_combine_gz)
```

Comment:
<3

Reply:
How do you feel about the naming? been on the verge of `RUN_<TOOL>` or just ` `<TOOL>` in the subworkflows directory. For now I dumped it all in the the folder `subworkflows/local/variantcalling` seems more cleaned up to me, but can also have one large subworkflow/local folder

Reply:
I'm happy with just `<TOOL>`.
Hoping to put it all in the nf-core/modules repo at some point, but that can wait the following releases

Reply:
yep that would be great. We would need to replace the concat script probably with the GATK concat tool or so (or find a good way to sharing a bin/ folder with each subworkflow)

Reply:
ok, just `<TOOL>` for the subworkflows doesn't work, if the modules have the same name. The subworkflow keeps trying to call itself then basically. I guess any process in a file is already "included" and takes priority over additional inclusions.